shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -387,16 +387,9 @@ public void sendTextInputAppPrivateCommand(@NonNull String action, @NonNull Bund
     mImm.sendAppPrivateCommand(mView, action, data);
   }
 
-  private boolean canShowTextInput() {
-    if (configuration == null || configuration.inputType == null) {
-      return true;
-    }
-    return configuration.inputType.type != TextInputChannel.TextInputType.NONE;
-  }
-
   @VisibleForTesting
   void showTextInput(View view) {
-    if (canShowTextInput()) {
+    if (configuration.inputType.type != TextInputChannel.TextInputType.NONE) {
```

Comment:
the old code is checking for configuration being null and inputType being null. Why don't we need those checks anymore?

Reply:
Added back.